data/json/flags.json

```
@@ -1363,6 +1363,10 @@
   {
     "id": "LEAK_DAM",
     "type": "json_flag"
+  },
+    {
```

Your JSON needs to be formatted:
```suggestion
  {
```